lib/v2/leetcode/today.js

```diff
@@ -0,0 +1,105 @@
+                variables: {
+                    questionSlug: questionTitle,
+                    first: 3,
+                    skip: 0,
+                    orderBy: 'DEFAULT',
+                },
+            },
+            headers,
+        })
+    ).data.data.questionSolutionArticles.edges;
+    // 获取题解内容
+    const articleContent = (
+        await Promise.all(
+            articles.map((art) =>
+                got({
+                    method: 'post',
+                    url,
+                    json: {
+                        operationName: 'solutionDetailArticle',
+                        query: 'query solutionDetailArticle($slug: String!, $orderBy: SolutionArticleOrderBy!) {\n  solutionArticle(slug: $slug, orderBy: $orderBy) {\n    ...solutionArticle\n    content\n    question {\n      questionTitleSlug\n      __typename\n    }\n    position\n    next {\n      slug\n      title\n      __typename\n    }\n    prev {\n      slug\n      title\n      __typename\n    }\n    __typename\n  }\n}\n\nfragment solutionArticle on SolutionArticleNode {\n  rewardEnabled\n  canEditReward\n  uuid\n  title\n  slug\n  sunk\n  chargeType\n  status\n  identifier\n  canEdit\n  canSee\n  reactionType\n  reactionsV2 {\n    count\n    reactionType\n    __typename\n  }\n  tags {\n    name\n    nameTranslated\n    slug\n    tagType\n    __typename\n  }\n  createdAt\n  thumbnail\n  author {\n    username\n    profile {\n      userAvatar\n      userSlug\n      realName\n      __typename\n    }\n    __typename\n  }\n  summary\n  topic {\n    id\n    commentCount\n    viewCount\n    __typename\n  }\n  byLeetcode\n  isMyFavorite\n  isMostPopular\n  isEditorsPick\n  hitCount\n  videosInfo {\n    videoId\n    coverUrl\n    duration\n    __typename\n  }\n  __typename\n}\n',
+                        variables: {
+                            slug: art.node.slug,
+                            orderBy: 'DEFAULT',
+                        },
+                    },
+                    headers,
+                })
+            )
+        )
+    ).map((v) => v.data.data.solutionArticle);
+    const converter = new showdown.Converter();
+    const handleText = (s) => {
+        // 处理多语言代码展示问题
+        s = s.replace(/(```)([a-zA-Z0-9-+#]+)\s*?(\[.*?\])?\n/g, '\r\n###$2\r\n $1$2\r\n');
+        return s;
+    };
+    ctx.state.data = {
+        title: 'LeetCode Today',
+        description: 'LeetCode 每日一题',
+        link: questionUrl,
+        item: [
+            {
+                title: `每日一题-${question.translatedTitle}`,
+                link: questionUrl,
+                description: question.translatedContent,
+                pubDate: data.todayRecord[0].date + ' 00:00:00',
+            },
+            ...articleContent.map((art, i) => ({
+                title: art.title,
+                link: `${questionUrl}/solution/${art.slug}`,
+                description: converter.makeHtml(handleText(art.content)),
+                pubDate: articles[i].node.createdAt,
```

Comment:
```suggestion
                pubDate: parseDate(articles[i].node.createdAt),
```
https://docs.rsshub.app/joinus/pub-date.html